### PR TITLE
chore(zh-cn): remove screenshots from EmbedLiveSample calls (Part 1)

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove screenshots from EmbedLiveSample calls

### Related issues and pull requests

Upstream: mdn/content#32505 ~(Blocked by: https://github.com/mdn/content/pull/32505#issuecomment-1979973621)~

### CODE REVIEW

File: files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md [hunk 1]
```diff
@@ -59,24 +67,24 @@ function init() {
...
   ctx.rotate(
     ((2 * Math.PI) / 60) * time.getSeconds() +
       ((2 * Math.PI) / 60000) * time.getMilliseconds(),
   );
   ctx.translate(105, 0);
-  ctx.fillRect(0, -12, 50, 24); // Shadow
+  ctx.fillRect(0, -12, 40, 24); // 阴影
   ctx.drawImage(earth, -12, -12);
 
   // Moon
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md:90]:
```suggestion
  // 月亮
```

File: files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md [hunk 2]
```diff
@@ -103,20 +111,27 @@ function draw() {
...
 
 ## 动画时钟
 
-这个例子实现一个动态时钟，可以显示当前时间。
+该示例绘制一个可以显示当前时间的动画时钟。
+
+### HTML
+
+```html
+<canvas id="canvas" width="150" height="150">The current time</canvas>
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md:125]:
```suggestion
<canvas id="canvas" width="150" height="150">当前时间</canvas>
```

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md [hunk 3]
```diff
@@ -67,22 +67,20 @@ function draw() {
...
-## 移动 Translating
+## 移动
 
-![](canvas_grid_translate.png)
-
-我们先介绍 `translate`方法，它用来移动 canvas 和它的原点到一个不同的位置。
+我们先介绍 `translate` 方法，它用来移动 canvas 和它的原点到一个不同的位置。
 
 - `translate(x, y)`
   - : `translate`方法接受两个参数。*x *是左右偏移量，_y_ 是上下偏移量，如右图所示。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:76]: canvas_grid_translate.png has not removed, it moved to here

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md [hunk 4]
```diff
@@ -107,7 +105,7 @@ function draw() {
 draw();
 ```
 
-{{EmbedLiveSample("translate 的例子", "160", "160", "translate.png")}}
+{{EmbedLiveSample("translate 的示例")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:108]:
```suggestion
{{EmbedLiveSample("translate 的示例", "", "160")}}
```

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md [hunk 5]
```diff
@@ -166,9 +164,9 @@ function draw() {
 draw();
 ```
 
-{{EmbedLiveSample("rotate 的例子", "310", "260", "rotate.png")}}
+{{EmbedLiveSample("rotate 的示例", "", "160")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:167]:
```suggestion
{{EmbedLiveSample("rotate 的示例", "", "220")}}
```

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md [hunk 6]
```diff
@@ -242,7 +240,7 @@ draw();
 - {{domxref("CanvasRenderingContext2D.resetTransform", "resetTransform()")}}
   - : 重置当前变形为单位矩阵，它和调用以下语句是一样的：`ctx.setTransform(1, 0, 0, 1, 0, 0);`
 
-### `transform`/`setTransform` 的例子
+### `transform`/`setTransform` 的示例
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:243]:
```suggestion
### `transform` 和 `setTransform` 的示例
```

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md [hunk 7]
```diff
@@ -273,6 +271,6 @@ function draw() {
 draw();
 ```
 
-{{EmbedLiveSample("transform/setTransform 的例子", "230", "280", "canvas_transform.png")}}
+{{EmbedLiveSample("transform/setTransform 的示例", "", "160")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:274]:
```suggestion
{{EmbedLiveSample("transform 和 setTransform 的示例", "", "260")}}
```

**Review by jasonren0403 — APPROVED** [review]
LGTM. Such a lot cleanups 🤔

File: files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md (hunk 3 above)
Comment (yin1999) [reply files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:76]:
```suggestion

![画布从网格上的原点水平向下向右平移“x”单位，垂直移动“y”单位。](canvas_grid_translate.png)

```

Comment (yin1999) [reply files/zh-cn/web/api/canvas_api/tutorial/transformations/index.md:76]: This has been remove from the upstream PR by mistake. But I forgot to add it back when the upstream has fixed this. Thanks for the reminder

**Pull request merged**